packages/framework/fluid-static/src/rootDataObject.ts

```diff
@@ -36,6 +39,14 @@ import {
 	parseDataObjectsFromSharedObjects,
 } from "./utils.js";
 
+/**
+ * Maps CompatibilityMode to a semver valid string that can be passed to the container runtime.
+ */
+const compatibilityModeToMinVersionForCollab: Record<CompatibilityMode, MinimumVersionForCollab> = {
+	"1": "1.0.0",
+	"2": "2.0.0",
+};
```

Comment:
(not a required change)
Following exact typing principles this is better written as:
```suggestion
const compatibilityModeToMinVersionForCollab = {
	"1": "1.0.0",
	"2": "2.0.0",
} as const satisfies Record<CompatibilityMode, MinimumVersionForCollab>;
```